### PR TITLE
Textfield making

### PR DESCRIPTION
**Pull request by filwas**

### CODE REVIEW

File: src/inputs/TextField/TextField.tsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+interface TextFieldProps {
+  /**
+   * If true, the input element is focused during the first mount.
+   */
+  autoFocus?: boolean;
+
+  /**
+   * Indicates whether the TextField is disabled or not.
+   */
```
Comment (Sumick) [inline src/inputs/TextField/TextField.tsx:16]: **praise:** Good job with describing props!

File: src/inputs/TextField/TextField.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+  /**
+   * Pleas use 24x24 px icons. 
+   */
+  icon?: React.ReactElement;
+}
+
+
+export const TextField = (props: TextFieldProps) => {
+  const [focused, setFocus] = useState(props.autoFocus ? true : false);
```
Comment (Sumick) [inline src/inputs/TextField/TextField.tsx:72]: **nitpick:** Why not ` const [focused, setFocus] = useState(props.autoFocus);` ?

File: src/inputs/TextField/TextField.tsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+  const labelErrorClass = !!props.error ? styles.labelError : "";
+  const disabledActiveClass = props.disabled ? styles.disabled : "";
+
+  const labelDynamicClasses = [
+    styles.baseLabel,
+    labelActiveClass,
+    labelFocusClass,
+    labelErrorClass,
+    disabledActiveClass,
+  ].join(" ");
```
Comment (Sumick) [inline src/inputs/TextField/TextField.tsx:91]: **suggestion:** You can create your own util function for joining classes, so you don't have to duplicate it every time you do this. Example: `const cx = (classNames: string[]) => classNames.join(" ")`. Btw, if you want to go step further, you can check out this lib - it makes working with class names much easier: https://github.com/JedWatson/classnames

**Review by Sumick — APPROVED** [review]
Good job! Im leaving 3 non-blocking comments 🙌

File: src/inputs/TextField/TextField.tsx (hunk 3 above)
Comment (filwas) [reply src/inputs/TextField/TextField.tsx:91]: This library looks cool, I'll use it in the next element.

**Pull request merged**